scripts/host-testbench.sh

```diff
@@ -11,6 +11,8 @@
 # ./scripts/host-testbench.sh
 #
 
+set -e
```

Comment:
Please add the same comments as below ' fail on errors'

Reply:
Done except I changed it to "stop on most errors". For legacy reasons `set -e` doesn't catch all errors, it catches most of them.